README.rst

```diff
@@ -778,6 +782,8 @@ The options (``[flags]``) are as follows:
 * ``--unit-params-file`` (used for ``rpm`` and ``deb`` packages) is is the path to the file that contains
   unit parameters for the ``systemd`` unit files.
 
+* ``--spec`` is the path to rockspec to use for current build.
```

Comment:
I think you need to place this flag in the same order as in ``cartidge pack --help`` (after ``--name`` flag)